packages/data-context/src/actions/EventCollectorActions.ts

```diff
@@ -1,7 +1,7 @@
 import type { DataContext } from '..'
 import Debug from 'debug'
 
-const debug = Debug('cypress:data-context:sources:EventCollectorDataSource')
+const debug = Debug('cypress:data-context:EventCollector')
```

Comment:
According to https://github.com/cypress-io/cypress/blob/develop/guides/debug-logs.md#choosing-a-namespace this would be

```suggestion
const debug = Debug('cypress:data-context:actions:EventCollectorActions')
```

which is verbose, but more predictable...

Reply:
Thanks @flotwig !